payu/models/cesm_cmeps.py

```diff
@@ -168,17 +168,16 @@ def setup(self):
 
         self.runconfig.write()
 
-        # TODO: Should have a better way to do this
+        # Horrible hack to make a link to the mod_def.ww3 input in the work
+        # directory
         # The ww3 mod_def input needs to be in work_path and called mod_def.ww3
         if "ww3dev" in self.components.values():
-            files =  glob.glob(
-                os.path.join(self.work_input_path, "*mod_def.ww3*"),
-                recursive=False
-            )
-            if files:
-                assert len(files) == 1, "Multiple mod_def.ww3 files found in input directory"
-                f_dst = os.path.join(self.work_path, "mod_def.ww3")
-                shutil.copy(files[0], f_dst)
+            f_name = "mod_def.ww3"
+            f_src = os.path.join(self.work_input_path, f_name)
+            f_dst = os.path.join(self.work_path, f_name)
```

Comment:
I like how straightforward this is. 

It will break some existing configs, so that should be communicated to those affected, but likely only a few people at this stage.

Just checking, there is no way to set the path of this file to be `input/mod_def.ww3`? That would be clearer, but I understand if this is hard-coded and not configurable then this is as good as it gets.

Reply:
> It will break some existing configs, so that should be communicated to those affected, but likely only a few people at this stage.

Already done: https://github.com/COSIMA/MOM6-CICE6-WW3/issues/2

> Just checking, there is no way to set the path of this file to be input/mod_def.ww3? That would be clearer, but I understand if this is hard-coded and not configurable then this is as good as it gets.

Not that Ezhil or I know of. But we're still learning about WW3. If I do discover it's possible, I'll fix this up.